tests/aws-cpp-sdk-dynamodb-integration-tests/CMakeLists.txt

```diff
@@ -13,7 +13,10 @@ if (NOT LEGACY_BUILD)
             ${CMAKE_CURRENT_SOURCE_DIR}/RunTests.cpp
             ${CMAKE_CURRENT_SOURCE_DIR}/TableOperationTest.cpp
     )
-    gtest_discover_tests(aws-cpp-sdk-dynamodb-integration-tests)
+    gtest_discover_tests(
+            aws-cpp-sdk-dynamodb-integration-tests
+            TEST_PREFIX "dynamodb."
```

Comment:
I guess this change shall be repeated for all client tests we have?

Reply:
Using DynamoDB as reference for the process, once DDB is 100% good and generated, I will expand to others.